### PR TITLE
Add support for default label on native environment repository

### PR DESCRIPTION
**Pull request by daniellavoie**
Brings support for `spring.cloud.config.server.defaultLabel` on the `NativeEnvironmentRepository`.

### CODE REVIEW

File: spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/NativeEnvironmentRepository.java [hunk 1]
```diff
@@ -49,13 +49,15 @@
...
 @ConfigurationProperties("spring.cloud.config.server.native")
-public class NativeEnvironmentRepository implements EnvironmentRepository, SearchPathLocator, Ordered {
+public class NativeEnvironmentRepository
+		implements EnvironmentRepository, SearchPathLocator, Ordered {
 
 	private static Log logger = LogFactory.getLog(NativeEnvironmentRepository.class);
-
-	private static final String DEFAULT_LABEL = "master";
+	
+	private String defaultLabel;
```
Comment (spencergibb) [inline spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/NativeEnvironmentRepository.java]: I think this still needs to default to `master`. If a user creates a `NativeEnvironmentRepository` bean, they wouldn't know they might need to update this.

Comment (daniellavoie) [reply spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/NativeEnvironmentRepository.java]: `DEFAULT_LABEL` was never used. Exposed through `getDefaultLabel` but never used.

Comment (spencergibb) [reply spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/NativeEnvironmentRepository.java]: right. `private String defaultLabel = "master"` is all I'm saying.

Comment (daniellavoie) [reply spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/NativeEnvironmentRepository.java]: In fact, the `profile` parameter will only be used if the users specify a custom location with `{label}` placeholder.

Updating to default to master.

Comment (daniellavoie) [reply spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/NativeEnvironmentRepository.java]: I guess if anyone relied on calling the public `getDefaultLabel()`, it's better to not break it. Fair enough.

**Pull request merged**